app/main.ipynb

```diff
@@ -129,18 +173,50 @@
+      "text/plain": [
+       "<Figure size 800x800 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "top_by_money = df.assign(total_price=df.quantity*df.price).groupby(\"name\", as_index=False).total_price.sum().sort_values(\"total_price\", ascending=False).head(10)\n",
-    "top_by_money.plot.pie(y=\"total_price\", labels=top_by_money.name, autopct=\"%1.1f%%\", title=\"Top 10 sold by value\", figsize=(8, 8))\n"
+    "top_by_money = (\n",
+    "    df.assign(total_price=df.quantity*df.price)\n",
+    "    .groupby(\"name\", as_index=False)\n",
+    "    .total_price.sum()\n",
+    "    .sort_values(\"total_price\", ascending=False).head(10)\n",
+    ")\n",
+    "top_by_money.plot.pie(\n",
+    "    y=\"total_price\",\n",
+    "    labels=top_by_money.name,\n",
```

Comment:
You can remove this label, you already have names written

Reply:
i've removed line with "labels" but now without it pie shows only indexes. Is it correct or i simply misunderstood smthng?

Reply:
Sorry for the unclear explanation 🤐 You should remove the legend to not provide information about colors. There was everything fine with labels, my bad